packages/shared/src/lib/core/account/actions/tryCreateAdditionalAccount.ts

```diff
@@ -21,11 +22,12 @@ export async function tryCreateAdditionalAccount(alias: string, color: string):
 
         const activeProfile = getActiveProfile()
         if (activeProfile.type === ProfileType.Software) {
-            const coinType = getEvmNetworks()[0]?.coinType
-            if (coinType !== undefined) {
-                void generateAndStoreEvmAddressForAccounts(activeProfile.type, coinType, account)
+            const coinType = DEFAULT_COIN_TYPE[SupportedNetworkId.Ethereum]
+            generateAndStoreEvmAddressForAccounts(activeProfile.type, coinType, account).then(async () => {
```

Comment:
I think that we could use `await` expression here

```suggestion
            await generateAndStoreEvmAddressForAccounts(activeProfile.type, coinType, account)
```

Reply:
after creating an account, we dont want to await the evm address generation. The `then` is used here because the methods in there rely on the evm address for the account.

But all in all, the `tryCreateAdditionalAccount` shouldnt be blocked for these functions to be finished